Dynatrace/dynatrace-operator
# Rename injected init-container

## Description
[DAQ-1583](https://dt-rnd.atlassian.net/browse/DAQ-1583)

We are not always injecting in the init container to install the oneagent, so to minimize confusion,  we are renaming it to something more generic

## How can this be tested?

Inject into sample apps, see that the init-container is renamed to `dynatrace-operator`.
An e2e test is also relied on this, so that should still work.

[DAQ-1583]: https://dt-rnd.atlassian.net/browse/DAQ-1583?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
